internal/lib/restore.go

```diff
@@ -3,8 +3,10 @@ package lib
 import (
```

Comment:
@adrianreber There are a few typos in the commit message. Would it be possible to fix them?